apps/wallet/src/ui/app/hooks/useDeepBook.ts

```diff
@@ -82,7 +100,7 @@ async function getDeepBookPriceForCoin(coin: Coins, deepbookClient: DeepBookClie
 		return 1n;
 	}
 
-	const poolName1 = `${coin}_USDC_1` as keyof typeof mainnetPools;
+	const poolName1 = `${coin}_USDC_1` as keyof (typeof mainnetDeepBook)['pools'];
```

Comment:
Why `_USDC_1` and `_USDC_2`? These names seem pretty confusing.

Reply:
There are 2 pools for SUI, we are checking both

Reply:
1. Why
2. Can we just structure this data in a better way? Like an array instead of flattening these into keyed structures